intg/src/main/java/org/apache/atlas/type/AtlasTypeRegistry.java

```diff
@@ -833,12 +842,15 @@ private void updateTypeWithNoRefResolve(AtlasBaseTypeDef typeDef) {
             }
         }
 
-        private void updateTypeByGuidWithNoRefResolve(String guid, AtlasBaseTypeDef typeDef) {
+        private void updateTypeByGuidWithNoRefResolve(String guid, AtlasBaseTypeDef typeDef) throws AtlasBaseException{
```

Comment:
Shouldn't we add validation in this "updateTypeByNameWithNoRefResolve" method as well?

Reply:
Added validation in "updateTypeByNameWithNoRefResolve" method and tested it locally and on beta.